### PR TITLE
LeetCode 75 Qs IssueNo.#453

### LINKED ISSUE #453: [General] LeetCode 75 Qs
The solution to the questions in Ace Coding Interview with 75 Qs in LeetCode

### PR DESCRIPTION
**Pull request by aditiverma-21**
### Description
I have All THE QUESTIONS OF ARRAY AND STRINGS in the present in practice of LEETCODE 75. Added more as discussed.

Fixes #453

### CODE REVIEW

**Comment by aditiverma-21** [comment]
@ajay-dhangar please merge the PR

**Comment by aditiverma-21** [comment]
@ajay-dhangar HELLO,
Please merge this PR!!

File: docs/75 LeetCode Questions/Arrays&Strings/4. Can Place Flowers.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+**Input:** flowerbed = [1,0,0,0,1], n = 2
+**Output:** false
+ 
+
+**Constraints:**
+- 1 <= flowerbed.length <= 2 * 104
+- flowerbed[i] is 0 or 1.
+- There are no two adjacent flowers in flowerbed.
+- 0 <= n <= flowerbed.length
+
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/4. Can Place Flowers.md]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/4. Can Place Flowers.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+id: can-place-flowers-605
+title:  Can Place Flowers
+sidebar_label: Leetcode 605
+tags: [Leetcode, String, DSA, Can Place Flowers]
+description: Given an integer array flowerbed containing 0's and 1's, where 0 means empty and 1 means not empty, and an integer n, return true if n new flowers can be planted in the flowerbed without violating the no-adjacent-flowers rule and false otherwise.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/4. Can Place Flowers.md]: add `"` for use description as  string

File: docs/75 LeetCode Questions/Arrays&Strings/5. Reverse Vowels of a String.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
+id: reverse-vowels-of-a-string-345
+title: Reverse Vowels of a String
+sidebar_label: Leetcode 345
+tags: [Leetcode, String, DSA, Reverse Vowels of a String]
+description: Given a string s, reverse only all the vowels in the string and return it.The vowels are 'a', 'e', 'i', 'o', and 'u', and they can appear in both lower and upper cases, more than once.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/5. Reverse Vowels of a String.md]: If you are using single quotes (`'`), then you need to write your string content in double quotes (`"`) with a close whole sentence. That way, the system can easily understand your code.

File: docs/75 LeetCode Questions/Arrays&Strings/5. Reverse Vowels of a String.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+The vowels in s are ['I', 'e', 'e', 'A']. On reversing the vowels, s becomes "AceCreIm".
+
+### Example 2:
+
+**Input:** s = "leetcode"
+**Output:** "leotcede"
+
+**Constraints:**
+- 1 <= s.length <= 3 * 105
+- s consist of printable ASCII characters.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/5. Reverse Vowels of a String.md]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/9.String Compression.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+Explanation: The only group is "a", which remains uncompressed since it's a single character.
+### Example 3:
+
+Input: chars = ["a","b","b","b","b","b","b","b","b","b","b","b","b"]
+Output: Return 4, and the first 4 characters of the input array should be: ["a","b","1","2"].
+Explanation: The groups are "a" and "bbbbbbbbbbbb". This compresses to "ab12".
+
+**Constraints:**
+
+1 <= chars.length <= 2000
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/9.String Compression.md:40]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/9.String Compression.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
+---
+id: string-compression-443
+title:  String Compression
+sidebar_label: Leetcode 443
+tags: [Leetcode, String, DSA,  String Compression]
+description: If the group's length is 1, append the character to s. Otherwise, append the character followed by the group's length. The compressed string s should not be returned separately, but instead, be stored in the input character array chars. Note that group lengths that are 10 or longer will be split into multiple characters in chars. After you are done modifying the input array, return the new length of the array. You must write an algorithm that uses only constant extra space.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/9.String Compression.md:6]: fix mistakes and we need on sort of way like a few words for SEO.

File: docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+### Example 3:
+
+**Input:** nums = [2,1,5,0,4,6]
+**Output:** true
+**Explanation:** The triplet (3, 4, 5) is valid because nums[3] == 0 < nums[4] == 4 < nums[5] == 6.
+ 
+**Constraints:**
+
+- 1 <= nums.length <= 5 * 105
+- -231 <= nums[i] <= 231 - 1
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+**Input:** nums = [5,4,3,2,1]
+**Output:** false
+**Explanation:** No triplet exists.
+### Example 3:
+
+**Input:** nums = [2,1,5,0,4,6]
+**Output:** true
+**Explanation:** The triplet (3, 4, 5) is valid because nums[3] == 0 < nums[4] == 4 < nums[5] == 6.
+
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md:28]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+id: increasing-triplet-subsequence-334
+title: Increasing Triplet Subsequence
+sidebar_label: Leetcode 334
+tags: [Leetcode, String, DSA, Increasing Triplet Subsequence]
+description: Given an integer array nums, return true if there exists a triple of indices (i, j, k) such that i < j < k and nums[i] < nums[j] < nums[k]. If no such indices exists, return false.
+---
+
+## 334. Increasing Triplet Subsequence
+**Description**: 
+Given an integer array nums, return true if there exists a triple of indices (i, j, k) such that i < j < k and nums[i] < nums[j] < nums[k]. If no such indices exists, return false.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+### Example 1:
+
+**Input:** nums = [1,2,3,4,5]
+**Output:** true
+**Explanation:** Any triplet where i < j < k is valid.
+### Example 2
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md:18]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
+---
+id: increasing-triplet-subsequence-334
+title: Increasing Triplet Subsequence
+sidebar_label: Leetcode 334
+tags: [Leetcode, String, DSA, Increasing Triplet Subsequence]
+description: Given an integer array nums, return true if there exists a triple of indices (i, j, k) such that i < j < k and nums[i] < nums[j] < nums[k]. If no such indices exists, return false.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/8. Increasing Triplet Subsequence.md]: Not a good way 👎

File: docs/75 LeetCode Questions/Arrays&Strings/7. Product of Array Except Self.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+### Example 2:
+
+**Input:** nums = [-1,1,0,-3,3]
+**Output:** [0,0,9,0,0]
+
+**Constraints:**
+
+- 2 <= nums.length <= 105
+- -30 <= nums[i] <= 30
+- The product of any prefix or suffix of nums is guaranteed to fit in a 32-bit integer.
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/7. Product of Array Except Self.md:28]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/7. Product of Array Except Self.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
+---
+id: product-of-array-except-self-238
+title: Product of Array Except Self
+sidebar_label: Leetcode 238
+tags: [Leetcode, String, Array, DSA, Product of Array Except Self]
+description: Given an integer array nums, return an array answer such that answer[i] is equal to the product of all the elements of nums except nums[i].
+---
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/7. Product of Array Except Self.md:7]: this is part of the props for our setup so, you need to use `key - value` pair and value as a string only and not use any programming keywords

File: docs/75 LeetCode Questions/Arrays&Strings/6. Reverse Words in a String.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+**Output:** "world hello"
+**Explanation:** Your reversed string should not contain leading or trailing spaces.
+###Example 3:
+
+**Input:** s = "a good   example"
+**Output:** "example good a"
+**Explanation:** You need to reduce multiple spaces between two words to a single space in the reversed string.
+
+**Constraints:**
+- 1 <= s.length <= 104
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/6. Reverse Words in a String.md]: replace `<` with `&lt;` and `>` with `&gt;`

File: docs/75 LeetCode Questions/Arrays&Strings/6. Reverse Words in a String.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
+---
+id: reverse-words-in-a-string-151
+title: Reverse Words in a String
+sidebar_label: Leetcode 151
+tags: [Leetcode, String, DSA, Reverse Words in a String]
+description: Given an input string s, reverse the order of the words.
+
+A word is defined as a sequence of non-space characters. The words in s will be separated by at least one space.
+---
```
Comment (ajay-dhangar) [inline docs/75 LeetCode Questions/Arrays&Strings/6. Reverse Words in a String.md:9]: write description only in a single line

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Update your code based on suggestions

File: docs/75 LeetCode Questions/Arrays&Strings/4. Can Place Flowers.md (hunk 1 above)
Comment (aditiverma-21) [reply docs/75 LeetCode Questions/Arrays&Strings/4. Can Place Flowers.md]: Yes sure

**Comment by aditiverma-21** [comment]
@ajay-dhangar resolved the problems Please check

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Checkout all files and change the code `<` and `>` because they are introducing as tags.

**Pull request closed without merging**